ajax/libs/equalize.js/package.json

```diff
@@ -13,7 +13,10 @@
   ],
   "homepage": "https://github.com/tsvensen/equalize.js",
   "licenses": [
-    "MIT",
+    {
```

Comment:
Please use `GPL-2.0` rather than modify `MIT` part. Thanks.

Reply:
@maruilian11 I think that got fixed in another branch of mine (equalize.js) https://github.com/cdnjs/cdnjs/pull/11988  I'm not sure why it's popping up in the branch intercom.js :thinking:

Reply:
Looks like you use a wrong branch(#11988) as base to create another branch for this PR, let's just change it back here, and we'll help you squash it.

Reply:
ok, done :)

Reply:
hmm ... looks not correct, @programmiri I think you should try to make the "diff" disappeared here in https://github.com/cdnjs/cdnjs/pull/11991/files